manifests/f/FACEITLTD/FACEITClient/1.31.11/FACEITLTD.FACEITClient.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Automatically updated by the winget bot at 2023/Jan/26
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: FACEITLTD.FACEITClient
+PackageVersion: 1.31.11
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://faceit-client.faceit-cdn.net/release/FACEIT-setup-latest.exe
+  InstallerSha256: B1FF742E204033DA41F7948C5AA291714CA959689221D6540E901C6FF428EBB9

```

Comment:
```suggestion
  InstallerSha256: 164d782441fc5fb8e97c47a8d1baf225d3ebda6403eaf56e53e1ba3b14497b5a
```

Reply:
It seems like the link's file has been reverted to the previous version. I've just redownloaded it and the hash is back to `b1ff742e204033da41f7948c5aa291714ca959689221d6540e901c6ff428ebb9`.